### PR TITLE
feat: support custom slug for find query param

### PR DESCRIPTION
**Pull request by mhatvan**
Closes https://github.com/amccloud/storybook-addons-abstract/issues/38

### CODE REVIEW

File: src/panel.tsx [hunk 1]
```diff
@@ -11,46 +11,47 @@ const ABSTRACT_APP_URL =
...
   const { storyId } = useStorybookState();
-  const params = useParameter<(Abstract.ShareDescriptor | Abstract.ShareUrlDescriptor) & {
-    options?: AddonOptions  
-  }>(
-    PARAM_KEY
-  );
+  const params = useParameter<
+    (Abstract.ShareDescriptor | Abstract.ShareUrlDescriptor) & {
+      options?: AddonOptions;
+      slug?: string;
```
Comment (amccloud) [inline src/panel.tsx:39]: This is great! My only suggestion is to call this `find` through and through

**Review by amccloud — COMMENTED** [review]
Hey hey thanks for the thorough report and PR. Overall the changes look okay and I made one suggestion

I'm not quite convinced we want to add this just yet. This feature was intentionally left out by design. The goal was for design and engineering teams to sync up on naming. The ideal scenario is a collection per component, with each state as an artboard in the collection

So with your original example `design-system-atoms-button--default`, you'd have a share link in the story to a collection called "So and So Button". Then add variants of your design system atoms button to that collection. You could then have a "Default" artboard and an artboard for each example in the story

`find` is a private query params, so it only really accepts what is on the right of `design-system-atoms-button--default` (e.g. `default`). With that, even if we do up needing a manual override for `find` in the end, I'll be happy to add it as long as we can document and test it 👍🏾

File: src/panel.tsx (hunk 1 above)
Comment (mhatvan) [reply src/panel.tsx]: I renamed "slug" to "find" in another commit ;)

**Comment by mhatvan** [comment]
It would be great to get a new release after this is merged, since we want to use this feature already, thanks! :rocket:

**Comment by mhatvan** [comment]
Hey @amccloud, can we get this PR merged?

**Comment by amccloud** [comment]
@mhatvan thanks for updating and pinging me. Will get this in 👍🏾

**Comment by mhatvan** [comment]
Perfect ;) WIll there be a new npm release?

**Comment by amccloud** [comment]
6.1.0 should be incoming now 👍🏾

**Pull request merged**